frontend/dev-server/index.js

```diff
@@ -7,18 +7,15 @@ const webpackDevMiddleware = require('webpack-dev-middleware');
 const webpackHotMiddleware = require('webpack-hot-middleware');
 const webpackHotServerMiddleware = require('webpack-hot-server-middleware');
 
-const { siteName, root } = require('./config');
+const { siteName, root, devServerPort } = require('../config');
 
 const go = async () => {
-    const webpackConfig = await require('./webpack');
+    const webpackConfig = await require('../../webpack');
```

Comment:
Although... seeing all this webpack stuff in here actually makes me think that this belongs under the top-level `/webpack` directory